recipes/binutils/all/conanfile.py

```diff
@@ -164,7 +164,7 @@ def yes_no(opt): return "yes" if opt else "no"
     def build(self):
         apply_conandata_patches(self)
         autotools = Autotools(self)
-        autotools.configure()
+        autotools.configure(args=["--sysconfdir=${prefix}/bin/etc"])
```

Comment:
Why does this need to be moved?

Reply:
because otherwise the linter complains that there is something in `etc`:
```
[HOOK - conan-center.py] post_package(): ERROR: [DEFAULT PACKAGE LAYOUT (KB-H013)] Unknown folder 'etc' in the package (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H013-DEFAULT-PACKAGE-LAYOUT)
```

And the file (`binut90310c647c75a/p/bin/etc/gprofng.rc`) in the "sysconfdir" looked useful:
```
# Specify which classes of compiler commentary will be shown
# with annotated source.
scc all

# Specify which classes of compiler commentary will be shown
# with annotated disassembly
dcc all:src

# Set the default function-list metrics
# for heap data, show inclusive leaks and bytes leaked; not allocations
dmetrics i.heapleakbytes:e!heapleakbytes
dmetrics i.heapleakcnt:e!heapleakcnt
dmetrics i.heapallocbytes:e!heapallocbytes
dmetrics i.heapalloccnt:e!heapalloccnt:
...
```

Reply:
Are you convinced ?

Reply:
@gjasny let me add an exception rule in the hook, so you don't need to change it. But first, why would need the etc folder? is there any important header there?

Reply:
[The file](https://github.com/bminor/binutils-gdb/blob/binutils-2_40/gprofng/src/gprofng.rc) seems to be a config file for the gprofng tool. It is read here: https://github.com/bminor/binutils-gdb/blob/32778522c7d8777803c88684b8e428ee729f0b22/gprofng/src/Settings.cc#L409-L413

Reply:
@gjasny Please,revert it. Let's use package_folder/etc instead. I'll open a PR to hooks, fixing the situation. Plus, I made some suggestions in cpp_info too, that are required in case using that file.